app/src/main/java/com/garan/wearwind/screens/SettingsDetail.kt

```diff
@@ -40,139 +42,90 @@ fun SettingsDetailScreen(
 
     LaunchedEffect(screenStarted) {
         if (screenStarted) {
             uiState.isShowTime.value = false
             uiState.isShowVignette.value = false
         }
     }
-    val constraintSet = ConstraintSet {
-        val minButton = createRefFor("minButton")
-        val maxButton = createRefFor("maxButton")
-        val minusButton = createRefFor("minusButton")
-        val plusButton = createRefFor("plusButton")
+    val currentValue = if (selected.value == SettingLevel.MIN) {
+        minMaxHolder.currentMin
+    } else {
+        minMaxHolder.currentMax
+    }.toFloat()
 
-        constrain(minButton) {
-            top.linkTo(parent.top)
-            bottom.linkTo(minusButton.top)
-            start.linkTo(parent.start)
-            end.linkTo(maxButton.start)
-        }
-        constrain(maxButton) {
-            top.linkTo(parent.top)
-            bottom.linkTo(plusButton.top)
-            start.linkTo(minButton.end)
-            end.linkTo(parent.end)
-        }
-        constrain(minusButton) {
-            top.linkTo(minButton.bottom)
-            bottom.linkTo(parent.bottom)
-            start.linkTo(parent.start)
-            end.linkTo(plusButton.start)
-        }
-        constrain(plusButton) {
-            top.linkTo(maxButton.bottom)
-            bottom.linkTo(parent.bottom)
-            start.linkTo(minusButton.end)
-            end.linkTo(parent.end)
-        }
-        createVerticalChain(
-            minButton, minusButton, chainStyle = ChainStyle.Packed
-        )
-        createVerticalChain(
-            maxButton, plusButton, chainStyle = ChainStyle.Packed
-        )
-        createHorizontalChain(
-            minButton, maxButton, chainStyle = ChainStyle.Packed
-        )
-        createHorizontalChain(
-            minusButton, plusButton, chainStyle = ChainStyle.Packed
-        )
+    val range = if (selected.value == SettingLevel.MIN) {
+        minMaxHolder.minRange()
+    } else {
+        minMaxHolder.maxRange()
     }
 
-    ConstraintLayout(constraintSet = constraintSet, modifier = Modifier.fillMaxSize()) {
-        val selected = remember { mutableStateOf(SettingLevel.MIN) }
-        Button(
-            modifier = Modifier
-                .layoutId("minButton")
-                .fillMaxSize(0.4f)
-                .padding(8.dp)
-                .then(
-                    if (selected.value == SettingLevel.MIN) {
-                        Modifier.border(2.dp, Colors.primary, CircleShape)
-                    } else Modifier
-                ),
-            colors = ButtonDefaults.secondaryButtonColors(),
-            onClick = {
-                selected.value = SettingLevel.MIN
-            }
+    Stepper(
+        value = currentValue,
+        onValueChange = { value ->
+            onClick(settingType, selected.value, value)
+        },
+        valueRange = range,
+        steps = ((range.endInclusive - range.start) / minMaxHolder.step).toInt() - 1
```

Comment:
A good candidate for a utility method in Wear Compose.  I can see us typing this a lot.  cc @bowersteve